app/main.py

```diff
@@ -1,25 +1,71 @@
 from abc import ABC
+from typing import Any
 
 
 class IntegerRange:
-    pass
+    def __init__(self, min_amount: int, max_amount: int) -> None:
+        self.min_amount = min_amount
+        self.max_amount = max_amount
+
+    def __set_name__(self, owner: Any, name: str) -> None:
+        self.protected_name = "_" + name
+
+    def __get__(self, obj: Any, objtype: Any = None) -> int:
+        return getattr(obj, self.protected_name)
+
+    def __set__(self, obj: Any, value: int) -> None:
+        if value in range(self.min_amount, self.max_amount + 1):
+            setattr(obj, self.protected_name, value)
+        else:
+            raise ValueError(f"value should be in the range from "
+                             f"{self.min_amount} to {self.max_amount}")
 
 
 class Visitor:
-   pass
+    def __init__(
+            self, name: str, age: int, weight: int, height: int,
+    ) -> None:
+        self.name = name
+        self.age = age
+        self.weight = weight
+        self.height = height
 
 
 class SlideLimitationValidator(ABC):
-    pass
+    def __init__(self, age: int, weight: int, height: int) -> None:
+        self.age = age
+        self.weight = weight
+        self.height = height
 
 
 class ChildrenSlideLimitationValidator(SlideLimitationValidator):
-    pass
+    age = IntegerRange(4, 14)
+    height = IntegerRange(80, 120)
+    weight = IntegerRange(20, 50)
+
+    def __init__(self, age: int, weight: int, height: int) -> None:
+        super().__init__(age=age, weight=weight, height=height)
```

Comment:
Redundant code. It works automatically
The same in AdultSlideLimitationValidator